projects/1.16.1/assets/playerex/playerex/lang/zh_cn.json

```
@@ -0,0 +1,108 @@
+	"playerex.config.common.strength": "力量",
+	"playerex.config.common.dexterity": "敏捷",
+	"playerex.config.common.intelligence": "智力",
+	"playerex.config.common.luckiness": "幸运",
+	"attribute.name.playerex.experience": "经验",
+	"attribute.name.playerex.level": "等级",
+	"attribute.name.playerex.skillpoints": "属性点",
+	"attribute.name.playerex.constitution": "体质",
+	"attribute.name.playerex.strength": "力量",
+	"attribute.name.playerex.dexterity": "敏捷",
+	"attribute.name.playerex.intelligence": "智力",
+	"attribute.name.playerex.luckiness": "幸运",
+	"attribute.name.playerex.health_regen": "生命恢复",
+	"attribute.name.playerex.health_regen_amp": "生命恢复倍率",
+	"attribute.name.playerex.damage_reduction": "伤害减免",
+	"attribute.name.playerex.fire_resistance": "火焰抗性",
+	"attribute.name.playerex.lava_resistance": "岩浆抗性",
+	"attribute.name.playerex.explosion_resistance": "爆炸抗性",
+	"attribute.name.playerex.falling_resistance": "摔落抗性",
+	"attribute.name.playerex.poison_resistance": "中毒抗性",
+	"attribute.name.playerex.wither_resistance": "凋零抗性",
+	"attribute.name.playerex.drowning_resistance": "溺水抗性",
+	"attribute.name.playerex.melee_crit_damage": "暴击伤害(近)",
+	"attribute.name.playerex.melee_crit_chance": "暴击机率(近)",
+	"attribute.name.playerex.evasion": "闪避",
+	"attribute.name.playerex.ranged_damage": "远程伤害",
+	"attribute.name.playerex.ranged_crit_damage": "暴击伤害(远)",
+	"attribute.name.playerex.ranged_crit_chance": "暴击机率(远)",
+	"attribute.name.playerex.lifesteal": "吸血",
+	"playerex.attribute.level": "等级: +%s (%s%s)",
+	"playerex.attribute.level.alt": "玩家等级,与附魔等级是分开的,每升级一级获得一点属性点.",
+	"playerex.attribute.skillpoints": "属性点: +%s",
+	"playerex.attribute.skillpoints.alt": "显示你还有多少属性点,用给增加下面那五个基础属性.",
+	"playerex.attribute.constitution": "体质: +%s",
+	"playerex.attribute.constitution.alt0": "+1 血量",
+	"playerex.attribute.constitution.alt1": "+1%% 击退抗性(递减)",
+	"playerex.attribute.strength": "力量: +%s",
+	"playerex.attribute.strength.alt0": "+0.02/秒 生命恢复",
+	"playerex.attribute.strength.alt1": "+0.25 近站伤害",
+	"playerex.attribute.strength.alt2": "+0.25 护甲(递减)",
+	"playerex.attribute.dexterity": "敏捷: +%s",
+	"playerex.attribute.dexterity.alt0": "+0.08 格/秒 移动速度(递减)",
+	"playerex.attribute.dexterity.alt1": "+5%% 暴击伤害(近)(递减)",
+	"playerex.attribute.dexterity.alt2": "+0.1 攻击速度(近)(递减)",
+	"playerex.attribute.dexterity.alt3": "+0.25 暴击伤害(远)",
+	"playerex.attribute.intelligence": "智力: +%s",
+	"playerex.attribute.intelligence.alt0": "+5%% 暴击伤害(远)(递减)",
+	"playerex.attribute.intelligence.alt1": "+2%% 吸血(递减)",
+	"playerex.attribute.intelligence.alt2": "+1%% 生命恢复与倍率(递减)",
+	"playerex.attribute.luckiness": "幸运: +%s",
+	"playerex.attribute.luckiness.alt0": "+1 运气.",
+	"playerex.attribute.luckiness.alt1": "+2%% 暴击机率(近)(递减)",
+	"playerex.attribute.luckiness.alt2": "+2%% 暴击机率(远)(递减)",
+	"playerex.attribute.luckiness.alt3": "+2%% 闪避(递减)",
+	"playerex.attribute.health": "血量: %s/%s",
+	"playerex.attribute.health.alt0": "当前生命 / 最大生命. \n生命恢复: 每秒恢复的生命. \n生命恢复倍率: 玩家每次恢复的额外增加的治疗量.",
```

|Feedback Priority|
|---|
|⏳ Sand / Non-blocking but requires future consideration|

```suggestion
	"playerex.attribute.health.alt0": "当前生命 / 最大生命 \n生命恢复: 每秒恢复的生命。 \n生命恢复倍率: 玩家每次恢复时额外增加的治疗量。",
```